src/phoenix/datasets/experiments.py

```diff
@@ -107,19 +102,27 @@ def _phoenix_client() -> httpx.Client:
     return client
 
 
+Evaluators: TypeAlias = Union[
+    ExperimentEvaluator,
+    Sequence[ExperimentEvaluator],
+    Mapping[EvaluatorName, ExperimentEvaluator],
```

Comment:
What do we get from this mapping input type? The only thing I can think is the ability to rename raw functions and coroutines to something other than their name, but we already provide a way to do that with `create_evaluator`.

Reply:
I think this adds an extra layer of convenience for the user and I don't see any downside to it. Yes, the decorator can do the same thing, but this is orthogonal because even if you have already applied the decorator you can still change its name if you just want test it or something (say you don't want to rerun the part of the code that defined the function). All in all, I think this is nice and adds only a trivial amount of complication.